### DISCUSSION TITLE
Error : failed to find the L2 Heads to start from: reorg is too deep

### DISCUSSION THREAD

**Discussion by dome** [ethereum-optimism/developers#495] [Application Development]
I try to test beta1 from replica script.
after docker compose down and up. got re-org error. fix seq_window_size in rollup.json not help.

goerli-op-node-1  | {"l1_head":"0x1e010ae9ce4847c2d85cd0e5cba28052ebb79f23f6476269e096fcb05d7c9219:8098832","lvl":"info","msg":"Received first L1 head signal","t":"2022-12-08T16:09:50.561443774Z"}
goerli-op-node-1  | {"attempts":3,"err":"stage 0 failed resetting: temp: failed to find the L2 Heads to start from: reorg is too deep: traversed back to L2 block 0xe2f6eff3b5e9ed276c1c91006b21c3a2a7752c154a665f4c7d8af3c820c29548:364763, but too deep compared to previous unsafe block 0xec8367050fb5fe6fc457a2e9345965ef6f76b46405665ea4843242d4d4079c6a:364962","lvl":"warn","msg":"Derivation process temporary error","t":"2022-12-08T16:09:53.124057479Z"}

**Comment by smartcontracts** [comment]
I think this would be better as an issue. Would you mind opening an issue instead with more information if possible?
